fireant/database/mysql.py

```diff
@@ -15,6 +15,11 @@ class Trunc(terms.Function):
     def __init__(self, field, date_format, alias=None):
         super(Trunc, self).__init__('dashmore.TRUNC', field, date_format, alias=alias)
 
+        # TODO fix pypika Function to expose kwargs (only args are exposed, which is more hacky to access).
```

Comment:
is this worth doing before merging this PR?

Reply:
Sure. I'll work on this tomorrow AM and update the PR.

Reply:
PyPika uses the order of the args in the Function class to generate the correct sql, and therefore it does not use the kwargs for the sql generation. Therefore, as this access is only required in Trunc right now, I'll leave it. We can re-visit it later if it becomes a pain point.